### PR TITLE
Update the direct award task list

### PR DESCRIPTION
**Pull request by lfdebrux**
This PR collects together a group of changes to the direct award task list that were put together by our UCD people.

Note that when these changes are merged, they will trigger a visual regression.

---

### Tickets in this PR
- [Change step 1 of the task list](https://trello.com/c/2SvOyqcv) @lfdebrux
- [Update content on Save search, Export search and existing tasklist](https://trello.com/c/hOL3asOK)
    - Update content on export search @lfdebrux
    - Update content on save search @galund

---

### Changes between this branch and master

master                           | epic-capture-spend-data-tasklist-changes
-----------------------|---------
![master_view-project] | ![e6b5c3b_view-project]
![master_results]          | ![740133e_results]

[master_view-project]: https://user-images.githubusercontent.com/503614/41777169-62d667c8-7622-11e8-8193-33b9f7e5f5af.png
[c75e8fa_view-project]: https://user-images.githubusercontent.com/503614/41777197-77a4a822-7622-11e8-8928-315ecf5368fd.png
[c74168d_view-project]: https://user-images.githubusercontent.com/503614/41908516-26bda450-793c-11e8-95d7-16cc66e2785b.png
[e6b5c3b_view-project]: https://user-images.githubusercontent.com/503614/41912442-b35c7890-7947-11e8-9861-ac484af1ce13.png
[master_results]: https://user-images.githubusercontent.com/503614/41905672-c74e9aae-7933-11e8-95bc-bfd82c07acc1.png
[740133e_results]: https://user-images.githubusercontent.com/503614/41905698-d59646f2-7933-11e8-9435-dbef66ce1dd9.png

### CODE REVIEW

File: app/templates/direct-award/view-project.html [hunk 1]
```diff
@@ -94,50 +80,50 @@
...
+                                                 data-analytics=\"trackEvent\"
+                                                 data-analytics-category=\"Direct Award\"
+                                                 data-analytics-action=\"External Link\"
+                                                >Customer Benefits Record form</a>
+                and email it to the Crown Commercial Service at <a href=\"mailto:", framework_urls.customer_benefits_record_form_email|e, "\"
+                                                                   target=\"_top\"
+                                                                   data-analytics=\"trackEvent\"
+                                                                   data-analytics-category=\"Direct Award\"
+                                                                   data-analytics-action=\"External Link\"
+                                                                  >", customer_benefits_record_form_email|replace("-", "&#8209;"|safe), "</a>"]
```
Comment (katstevens) [inline app/templates/direct-award/view-project.html]: Is this missing the `framework_urls.` or are we declaring `customer_benefits_record_form_email` somewhere else in the context? 

Also, I'd probably split this string up for readability, seeing as we're joining it anyway.

```
''.join([
    "You must fill in the",
    " <a href=link with all that analytics stuff>Customer Benefits Record form</a>",
    " and email it to CCS at ",
    " <a href=...>",
    framework_urls.customer_benefits_record_form_email|replace(...),
    "</a>"
])
```

File: app/templates/direct-award/save-search.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {% extends "toolkit/layouts/_base_page.html" %}
 
-{% block page_title %}Choose where to save your search - Digital Marketplace{% endblock %}
+{% set page_name = "Save your search" %}
```
Comment (katstevens) [inline app/templates/direct-award/save-search.html]: If we're doing this here, should we do it for `Download your results` as well?

File: app/assets/scss/application.scss [hunk 3]
```diff
@@ -101,3 +101,19 @@ $path: "/static/images/";
   display: inline-block;
   word-break: break-all;
 }
+
+.indent {
```
Comment (galund) [inline app/assets/scss/application.scss]: We think we can remove in favour of `application-notice info-notice`

**Comment by karlchillmaid** [comment]
**Task list content changes**
As the task list we're going to release combines elements of the old and new, I think it might need a couple of additional content changes to smooth over the gaps.

`2. Export your results`
`Export your search results to keep a record of the services you’ve found. You should only complete this step when you're ready to start assessing services.`

`3. Download and assess your results`
`Download a spreadsheet of suppliers’ service descriptions and contact details. Follow the [guidance for assessing services]. Do not hold a competition to decide the winner.`
link [https://www.gov.uk/guidance/g-cloud-buyers-guide#review-and-compare-services]

**Pull request merged**